core/src/com/mygdx/insanerpg/model/unit/Player.java

```diff
@@ -0,0 +1,62 @@
+package com.mygdx.insanerpg.model.unit;
+
+abstract class Player extends Unit implements Movable{
+    private int _intelligence;
+    private int _dexterity;
+    private int _vitality;
+    private int _strength;
+
+    public void Move(double cursorXCoordinate, double cursorYCoordinate) {
```

Comment:
this will technically not work since IE. you are at 0,0 but you click 1,1 but you move .3 every tick, it will skip over the exact 1,1 and keep moving back and forth of where you clicked. Try making the cursorX and cursorY a range of pixels near where you clicked.